src/jvmMain/kotlin/net/kyori/adventure/webui/jvm/minimessage/MiniMessage.kt

```diff
@@ -22,9 +23,24 @@ import net.kyori.adventure.webui.jvm.appendComponent
 import net.kyori.adventure.webui.jvm.getConfigString
 import net.kyori.adventure.webui.jvm.minimessage.editor.installEditor
 import net.kyori.adventure.webui.jvm.minimessage.hook.*
-import net.kyori.adventure.webui.websocket.Call
-import net.kyori.adventure.webui.websocket.ParseResult
-import net.kyori.adventure.webui.websocket.Response
+import net.kyori.adventure.webui.websocket.*
+
+public fun Templates.templateResolver(): TemplateResolver {
```

Comment:
This could be a val rather than a function

Reply:
extension getter...?